irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -1197,6 +1224,44 @@ namespace iroha {
           sql_, cmd.str(), "TransferAsset", std::move(str_args));
     }
 
+    CommandResult PostgresCommandExecutor::operator()(
+        const shared_model::interface::CompareAndSetAccountDetail &command) {
+      auto &account_id = command.accountId();
+      auto &key = command.key();
+      auto &value = command.value();
+      auto &old_value = command.oldValue();
+
+      if (creator_account_id_.empty()) {
+        // When creator is not known, it is genesis block
+        creator_account_id_ = "genesis";
```

Comment:
This looks like a potential vulnerability.
Under some circumstances, someone could mimic genesis user and get a set of elevated privileges.

Maybe this is not possible, but up to that point, this looks suspicious.

Reply:
So I removed it, but the same construction used in ```SetAccountDetail``` command

Reply:
Fixed